syftr/prompts/judge.py

```diff
@@ -86,7 +86,7 @@
 }
 """
 
-JUDGE_SYSTEM_PROMPT_SIMPLE: str = "Return YES if the Generated Answer is correct relative to the Reference Answer, or NO if it is not."
+JUDGE_SYSTEM_PROMPT_SIMPLE: str = "Return *YES* if the Generated Answer is correct relative to the Reference Answer, or *NO* if it is not. Make sure to use asterisks around the answer and return only *YES* or *NO*."
```

Comment:
Do you have a study showing the impact of this? I'd be concerned that models would output only `YES` or `NO`. And do you have evidence that the previous version was getting incorrect results?

Reply:
We still need to generate and compare data. I feel parsing for just YES is too general and should give wrong signals.

Reply:
Random checks of log file show only sporadic issues with non-simple prompts of the type "Could not parse score from response". Simple prompt parsing seems to work fine. There are no failed trials overall.